utilities/logistics-b2b/log-verification-utility/schema/B2B_json_schema/v2/on_update.js

```diff
@@ -0,0 +1,530 @@
+                      },
+                      settlement_bank_account_no: {
+                        type: "string",
+                      },
+                      settlement_ifsc_code: {
+                        type: "string",
+                      },
+                      bank_name: {
+                        type: "string",
+                      },
+                      branch_name: {
+                        type: "string",
+                      },
+                    },
+                    allOf: [
+                      {
+                        if: {
+                          properties: {
+                            settlement_type: {
+                              const: "upi",
+                            },
+                          },
+                        },
+                        then: {
+                          required: ["upi_address"],
+                        },
+                      },
+                      {
+                        if: {
+                          properties: {
+                            settlement_type: {
+                              const: ["neft", "rtgs"],
+                            },
+                          },
+                        },
+                        then: {
+                          required: [
+                            "settlement_ifsc_code",
+                            "settlement_bank_account_no",
+                          ],
+                        },
+                      },
+                    ],
+                    required: ["settlement_counterparty", "settlement_type"],
+                  },
+                },
+              },
+              if: { properties: { type: { const: "ON-FULFILLMENT" } } },
+                then: {
+                  properties: {
+                    collected_by: {
+                      const: "BPP",
+                    },
+                  },
+                },
+              required: [
+                "params",
+                "status",
+                "type",
+                "collected_by",
+                "@ondc/org/buyer_app_finder_fee_type",
+                "@ondc/org/buyer_app_finder_fee_amount",
+              ],
+            },
+            fulfillments: {
+              type: "array",
+              items: {
+                type: "object",
+                properties: {
+                  id: {
+                    type: "string",
+                  },
+                  "@ondc/org/provider_name": {
+                    type: "string",
+                  },
+                  state: {
+                    type: "object",
+                    properties: {
+                      descriptor: {
+                        type: "object",
+                        properties: {
+                          code: {
+                            type: "string",
+                            enum:["Pending","Agent-assigned","Order-picked-up","Out-for-delivery","Delivered"]
```

Comment:
Pending, Out-for-pickup, Order-picked-up, In-transit, At-destination-hub, Out-for-delivery, Order-delivered